package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "johnny-five",
-  "description": "The JavaScript Robotics and Hardware Programming Framework. Use with: Arduino (all models), Electric Imp, Beagle Bone, Intel Galileo & Edison, Linino One, Pinoccio, pcDuino3, Raspberry Pi, Particle/Spark Core & Photon, Tessel 2, TI Launchpad and more!",
-  "version": "2.1.0",
+  "name": "@code-dot-org/johnny-five",
+  "description": "Code.org fork of rwaldron/johnny-five: The JavaScript Robotics and Hardware Programming Framework. Use with: Arduino (all models), Electric Imp, Beagle Bone, Intel Galileo & Edison, Linino One, Pinoccio, pcDuino3, Raspberry Pi, Particle/Spark Core & Photon, Tessel 2, TI Launchpad and more!",
+  "version": "0.11.1-cdo.2",
```

Comment:
Does the version here matter? I know we're not using this package, but are there potential impacts on anyone who has forked our main code-dot-org repo (and still relies on this package)?

Side note, how did you find the "code.org customizations" you re-added in your last couple commits?

Reply:
Yes, I think the version does matter! Thanks for bring this up. I was thinking that we would complete the updates laid out in [this investigation](https://codedotorg.atlassian.net/browse/SL-14) and then make a commit to mark the release of 2.1.0-cdo.3. But if there's another convention we want to use, I'm definitely open to it! I'll add this to the summary above.

Also, I found the code.org-specific verbiage by lookin at the diffs when I reset the head of my branch to the latest release from [upstream](https://github.com/rwaldron/johnny-five). Lmk if there's a better way to handle this in the future. Thanks so much!